qa/build.xml

```diff
@@ -84,45 +78,51 @@
     <mkdir dir="${org.jboss.jbossts.qa.dist.buildroot}"/>
     <mkdir dir="${org.jboss.jbossts.qa.ts.home}"/>
-    <unzip src="../narayana-full/target/narayana-full-${narayana.version}-bin.zip" dest="${org.jboss.jbossts.qa.ts.home}"/>
-    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-idlj.jar" tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-openjdk.jar"/>
+    <unzip src="../narayana-full/target/narayana-full-${narayana.version}-bin.zip"
+           dest="${org.jboss.jbossts.qa.ts.home}"/>
+    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-idlj.jar"
+          tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-openjdk.jar"/>
     <antcall target="init-jta"/>
     <antcall target="init-jts"/>
   </target>
 
   <target name="init-jta" if="init-jta">
-    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jta/narayana-jta.jar" tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/narayana-jta.jar"/>
+    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jta/narayana-jta.jar"
+          tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/narayana-jta.jar"/>
   </target>
 
   <target name="init-jts" depends="jts.cp"/>
 
   <target name="jts.cp">
     <echo>copying ${orbtype} jts</echo>
-    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-${orbtype}.jar" tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jbossjts.jar"/>
+    <copy file="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jts/narayana-jts-${orbtype}.jar"
+          tofile="${org.jboss.jbossts.qa.ts.home}/narayana-full-${narayana.version}/lib/jbossjts.jar"/>
   </target>
 
   <target name="dist" depends="clean, init, buildtype">
     <!-- build the test suite -->
     <ant dir="tests" antfile="build-${buildtype}.xml">
       <property name="orbtype" value="${orbtype}"/>
-      <property name="org.jboss.jbossts.qa.server_manager_location" value="${org.jboss.jbossts.qa.server_manager_location}"/>
+      <property name="org.jboss.jbossts.qa.server_manager_location"
+                value="${org.jboss.jbossts.qa.server_manager_location}"/>
     </ant>
   </target>
 
   <target name="help">
-    <echo>Supported targets:
-  jta - build tests for the standalone jta transaction engine
-  jts - build tests for the standalone jts transaction engine
-  narayana-jta - build tests for the integrated jta transaction engine
-  narayana-jts - build tests for the integrated jts transaction engine
-  dist - build tests for the last build of the transaction engine
-</echo>
+    <echo>
+      Supported targets: jta - build tests for the standalone jta transaction engine jts - build tests for the
```

Comment:
This formatting seems a bit weird to me. Should not be targets printed on a separate line?

Reply:
The previous formatting was in style with end of lines after each target listed, i.e.
```
Supported targets:
  jta - build tests for the standalone jta transaction engine 
  jts - build tests for the standalone jts transaction engine
  ...
```

while your proposed change is about having it somehow mixed
```
Supported targets: jta - build tests for the standalone jta transaction engine jts - build tests for the
  standalone jts transaction engin
  ...
```
I mean every target `jta`,`jts`,`narayana-jta`,`narayana-jts`,`dist` should be placed on a separate line.

Reply:
Oh, I see now! Thank you very much for the explanation @ochaloup  +1